contracts/reflect/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 pub mod contract;
+mod errors;
```

Comment:
A bit interesting if we use `pub mod`, but only when we expect the contract logic to be directly imported by other contracts.

(Just thinking aloud here)

Reply:
I don't think contract specific errors should ever be public since they are fully crate-internal.